database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/source/DBSource.java

```diff
@@ -44,34 +45,38 @@
 import java.sql.Driver;
 
 /**
- * Batch source to read from a Database table
+ * Batch source to read from a DB table
  */
 @Plugin(type = "batchsource")
 @Name("Database")
-@Description("Reads from a database using a configurable SQL query." +
+@Description("Reads from a database table using a configurable SQL query." +
```

Comment:
this could be misleading, as somebody can use an import query that reads from multiple tables